recipe/meta.yaml

```diff
@@ -40,23 +49,22 @@ requirements:
     - clangxx                # [win]
     - python
     - {{ pin_compatible('numpy') }}
-    - decorator
-    - gast 0.4
-    - networkx >=2
+    - gast >=0.5.0,<0.6.0
```

Comment:
while the requirement for `gast` will work the upstream does not seem to give a maximum versioon limit for `gast` on version 0.10.0 Just curious if there is any benefit on restricting the uper pinning

https://github.com/serge-sans-paille/pythran/blob/0.10.0/requirements.txt#L2

Reply:
> It seems that to support `python 3.10` we probably might want to also allow for higher versions of the dependencies
> 
> https://github.com/serge-sans-paille/pythran/blob/0.10.0/Changelog#L18

I've already bumped pinning fro gast and beniget